src/entry.c

```diff
@@ -436,7 +456,7 @@ size_t entryMemUsage(entry *entry) {
          * header could be too small for holding the real allocation size. */
         mem += zmalloc_usable_size(entryGetAllocPtr(entry));
     }
-    mem += sdsAllocSize((sds)entryGetValue(entry, NULL));
+    if (!entryHasStringRef(entry)) mem += sdsAllocSize((sds)entryGetValue(entry, NULL));
```

Comment:
I still feel like we miss accounting for the stringRef (16 bytes) itself.

Reply:
This is accounted for in line 457.

Reply:
@yairgott maybe the lines count shifted with some of the changes. 
If I follow the memory count steps, the memory with stringRef dose NOT have an embedded value so I am taking the malloc size of the main entry buffer:
    
      |  long long   | stringRef (pointer) | sdshdr8+ | "foo" \0 |/ / / / |

Now, that include the pointer to the stringref mid-layer, but we never add this to the count. 

```suggestion
    if (!entryHasStringRef(entry)) mem += sdsAllocSize((sds)entryGetValue(entry, NULL));
    else mem += sizeof(stringRef); // can also consider mem += zmalloc_usable_size(entryGetStringRefRef(entry));
```

Reply:
Thanks for the clarification!